models/concerns/validate_data.rb

```diff
@@ -27,6 +27,8 @@ def check_consistency(hash)
     hash[:relations] ||= []
     @hash = hash
 
+    ensure_no_conflicting_skills
+
```

Comment:
Is this correct? It looks like a function not related to model solution dev.

Reply:
:+1:  It should not be there